### ISSUE TITLE
[Jira] Extend index to track assignees

### ISSUE THREAD

**Issue by alpgarcia** [chaoss/grimoirelab-elk#656]
Despite of assignee information is included in current index, it is not possible to create a single table like:

Name | Tasks Created  | Tasks Assigned
----|----|-----
John Doe | 3 | 2 

The reason is the name bucket needs to be common, that is, using the same field. If we use `author_name`, then we cannot count assigned tasks as each item in the index is a task and `author_name` is the author of the creator.

To build a table like above, we would need to store a specific item with a different type, so `author_name` contains assignee name. This way, for a ticket created by `John Doe` and assigned to `Joan Dee` we would store two documents in ES:
```
"Item 1": {
    "type": "issue",
    "author_name": "John Doe",
    ...
}
```
```
"Item 2": {
    "type": "assigned_issue",
    "author_name": "Joan Dee",
    ...
}
```
Some implications I see:
* For most metrics, we would need to exclude items of type `assigned_issue`. That means we would need to update Sigils panels to be compatible with this change.
* Each item would be stored twice, this doesn't seem to be a problem, but...

Any other proposals are welcome, this is just mine :)

**WARNING**: Sigils Jira panels must be updated accordingly before closing this task.

**Comment by valeriocos** [comment]
Issue addressed by https://github.com/chaoss/grimoirelab-elk/pull/659

**Comment by alpgarcia** [comment]
Thanks, I'll test it!

**Comment by sduenas** [comment]
Already implemented by #659
